core/modules/cms/store/block/actions.ts

```diff
@@ -11,6 +11,7 @@ const actions: ActionTree<CmsBlockState, RootState> = {
       const blockResponse = await quickSearchByQuery({
         query: createLoadingBlockQuery({ filterField, filterValues }),
         entityType: 'cms_block',
+        size,
```

Comment:
you could have added start too ;)

Reply:
done, have to change IDE theme again :D because I thought everything was marked